src/app/send/send.component.html

```diff
@@ -105,7 +105,7 @@ <h2>{{ "types" | i18n }}</h2>
       <div class="no-items" *ngIf="!filteredSends.length">
         <i class="fa fa-spinner fa-spin fa-3x" *ngIf="!loaded" aria-hidden="true"></i>
         <ng-container *ngIf="loaded">
-          <i class="fa fa-frown-o fa-4x" aria-hidden="true"></i>
+          <img class="no-items-image" />
```

Comment:
Please ensure we're providing alt-text, a title description, aria-* helpers or making this image `aria-hidden="true"` since it wouldn't actually help the visually impaired contextually (since there is also already text conveying this meaning and we would want to avoid redundancy)

Reply:
Will do!

Reply:
Added `aria-hidden="true"` as there is already text conveying the message